projects/iio_demo/src/parameters.h

```diff
@@ -86,10 +83,12 @@
 #define UART_DEVICE_ID	XPAR_AXI_UART_DEVICE_ID
 #define INTC_DEVICE_ID	XPAR_INTC_SINGLE_DEVICE_ID
 #define UART_IRQ_ID		XPAR_AXI_INTC_AXI_UART_INTERRUPT_INTR
-#define UART_BAUDRATE	921600
-
 #endif // _XPARAMETERS_PS_H_
 
+/* 400 * 8 * 2 = 6400‬ Default number of samples requested on a capture */
+#define MAX_SIZE_BASE_ADDR	10000
+#define UART_BAUDRATE	921600
```

Comment:
The baud rate of the AXI UartLite (used in our MicroBlaze designs), is not software configurable - it can be changed only through Vivado. By default, it is set to 115200.

Reply:
done